content/en/blog/_posts/2017-05-00-Kubernetes-Monitoring-Guide.md

```diff
@@ -3,6 +3,8 @@ title: " Kubernetes: a monitoring guide "
 date: 2017-05-19
 slug: kubernetes-monitoring-guide
 url: /blog/2017/05/Kubernetes-Monitoring-Guide
+author: >
+  Jean-Mathieu Saponaro (Datadog)
 ---
 _Today’s post is by Jean-Mathieu Saponaro, Research & Analytics Engineer at Datadog, discussing what Kubernetes changes for monitoring, and how you can prepare to properly monitor a containerized infrastructure orchestrated by Kubernetes._  
```

Comment:
I'd drop this note. We now just credit the person and optionally their organization.